README.md

```diff
@@ -67,27 +57,81 @@
+
+<p align="center">
+    <img width="800" src="images/qualcomm-image1.png">
+</p>
+<p align="center">Sign-Up</p>
+
+### Create a Factory
+
+1. Select the platform \*
+2. Name your Factory
+3. Click the Create Factory button
+
+_\* If you want to try FoundriesFactory on a different Qualcomm platform,
+create the Factory as suggested for Qualcomm RB3 Gen 2 Development Kit and contact Foundries.io at contact@foundries.io._
+
+<p align="center">
+    <img width="800" src="images/qualcomm-image2.png">
+</p>
+<p align="center">Platform Selection</p>
+
+NOTE: Based on the selected platform, the `<machine-name>` changes as shown in the table below:
+
+| Device Name | `<machine-name>` |
+| ------ | ------ |
+| Qualcomm RB3 Gen 2 | ``qcm6490`` |
+
+### Members Tab
 
-1. To start a new Factory, please visit https://app.foundries.io/ and sign up for an account.
-2. In the Platform drop down, select Qualcomm RB3 Gen 2 Development Kit.
-3. Give your Factory a name, and click "Create Factory".
-4. The Factory owners may invite additional users to their FoundriesFactory Account via email under the "Members" tab of the Factory interface.
+The Factory owners may invite additional users to their account via email under the "Members" tab of the Factory interface.
     - Manage your Factory users: ```https://app.foundries.io/factories/<factory>/members/```
-5. Fioctl CLI Installation: https://docs.foundries.io/latest/getting-started/install-fioctl/index.html
-6. Configure git using Fioctl: https://docs.foundries.io/latest/getting-started/git-config/index.html
 
-**Note**: On macOS, you may encounter authentication issues due to Git on OSX using the Keychain Access Utility. The solution is to remove Keychain Access entries from your git config file.
+### Fioctl Installation
 
+Fioctl™ is a simple tool for interacting with the Foundries.io™ REST API.
+- [Fioctl CLI Installation](https://docs.foundries.io/latest/getting-started/install-fioctl/index.html#gs-install-fioctl)
+
+### Configure git
+
+After Fioctl is properly setup, use its Git credential helper to allow pushing to your repos with FoundriesFactory.
+- [Configuring Git](https://docs.foundries.io/latest/getting-started/git-config/index.html#gs-git-config)
+
+**Note**: On macOS, you may encounter authentication issues due to Git on OSX using the Keychain Access Utility.
+The solution is to remove Keychain Access entries from your git config file.
 
 ### Qualcomm® Robotics RB3G2 Development Kit
 
-Once your Factory has been created, it will build the source code for the RB3G2 and produce a Target. A Target is a secure over-the-air update but also provides the build artifacts for initial provisioning.
+Once creating your Factory, it will build the source code for the RB3G2 and produce a Target.
+A Target is a over-the-air update that can provide the build artifacts for initial provisioning.
+
+1. Navigate To the Targets Section of Your Factory
+Click the latest Targets with the platform-devel Trigger.
```

Comment:
I believe the default platform is now platform-main.

Reply:
Good catch, it's been that way for a good bit now...this must of have generated from some old documentation.